src/pages/data.tsx

```diff
@@ -0,0 +1,199 @@
+    text: "their clever design features help keep them slim, even when they're full…",
+  },
+  {
+    url: "https://bellroy-cms-images.imgix.net/4425/slider-image04.jpg",
+    text: "so they'll be with you from Day One to Day 1000, and beyond.",
+  },
+];
+
+
+export const whyDoYouCarryContent = [
+  {
+    text: "Cards only",
+    inactiveImage: "/icon-inactive-card-sleeve.jpg",
+    activeImage: "/icon-active-card-sleeve.avif",
+  },
+  {
+    text: "Cards and bills",
+    inactiveImage: "/icon-inactive-card-sleeve.jpg",
+    activeImage: "/icon-active-note-sleeve.jpg",
+  },
+  {
+    text: "Cards, bills and coins",
+    inactiveImage: "/icon-inactive-folio.jpg",
+    activeImage: "/icon-active-folio.avif",
+  },
+];
+
+export const cardsOnlyProduct = [
+  {
+    name: "Card Sleeve",
+    colours: [
+      "black",
+      "card-sleeve-ocean",
+      "card-sleeve-gray",
+      "card-sleeve-teal",
+      "card-sleeve-hazel",
+    ],
+    price: "C$69",
+    image: [
+      "/card-Sleeves/card-Sleeve_Black.avif",
+      "/card-Sleeves/card-Sleeve_Ocean.avif",
+      "/card-Sleeves/card-Sleeve_Gray.avif",
+      "/card-Sleeves/card-Sleeve_Teal.avif",
+      "/card-Sleeves/card-Sleeve_Hazel.avif",
+    ],
+  },
+  {
+    name: "Card Sleeve",
+    colours: ["black"],
+    price: "C$69",
+    image: ["/card-Sleeves/card-Sleeve-Black_Ash.avif"],
+    edition: "Carryology Essentials Edition",
+  },
+  {
+    name: "Flip Case",
+    colours: ["black"],
+    price: "C$69",
+    image: ["/card-Sleeves/flip-Case_Terracotta.avif"],
+    edition: "Second Edition",
+  },
+  {
+    name: "Card Pocket",
+    colours: ["black"],
+    price: "C$85",
+    image: ["/card-Sleeves/card-Pocket-Ranger_Green.avif"],
+  },
+  {
+    name: "Phone Case - 3 Card",
+    colours: ["black"],
+    price: "C$99",
+    image: ["/card-Sleeves/phone-Case_Black.avif"],
+  },
+  {
+    name: "Card Sleeve",
+    colours: ["black"],
+    price: "C$95",
+    image: ["/card-Sleeves/card-Sleeve_Mirum_Black.avif"],
+    edition: "MIRUM Edition",
+  },
+];
+
+export const cardsAndBillsProduct = [
+  {
+    name: "Apex Slim Sleeve",
+    colours: ["black", "card-sleeve-gray", "card-sleeve-ocean"],
+    price: "C$165",
+    image: [
+      "/wallets-coins/apex-Slim-Sleeve_Raven.avif",
+      "/CB-1.avif",
+      "/CB-2.avif",
+    ],
+    edition: "RFID safe",
+  },
+  {
+    name: "Card Sleeve",
+    colours: ["black"],
+    price: "C$99",
+    image: ["/CB-3.avif"],
+    edition: "Carryology Essentials Edition",
+  },
+  {
+    name: "Slim Sleeve",
+    colours: [
+      "black",
+      "card-sleeve-gray",
+      "card-sleeve-ocean",
+      "card-sleeve-teal",
+      "wallet-chocolate",
+      "card-sleeve-hazel",
+    ],
+    price: "C$99",
+    image: [
+      "/CB-4.avif",
+      "/CB-6.avif",
+      "/CB-7.avif",
+      "/CB-8.avif",
+      "/CB-10.avif",
+      "/CB-11.avif",
+    ],
+  },
+  {
+    name: "Minimalist Set",
+    colours: ["black"],
+    price: "C$109 - $C129",
```

Comment:
This is less of a change but becareful with doing stuff like this. This isn't how you would want to display certain prices. 

Can you tell me why we wouldn't want to do this?

Don't resolve this after answering, I'll resolve this one when we come to an understanding

Reply:
the sequencing is incorrect as well, if we want to have different currency for the future it would extra code remove the C's from the string. It's better to write the price range is "$109 - $129" and have a separate array for the currency sign based on other parameters.

Reply:
That's a good point, currency conversion would be a pain. but another pain to consider is that you wouldn't want to have ranges like this in a string. What happens when you want to change it? I think have a min price and max price would make more sense and then structure your code to read either a single price or a price range.